nmrfx-processor/src/main/java/org/nmrfx/processor/optimization/LmdifTest_f77.java

```diff
@@ -313,27 +313,6 @@ public void doMin() {
             //lmdiftest.nfev = 0;
             Minpack_f77.lmdif1_f77(lmdifFunc, m, n, a, fvec, tol, info);
             lmdifFunc.fcn(m, n, a, fvec, iflag);
-
-            //System.out.println("\n Initial L2 norm of the residuals: " + fnorm1 +
-            //"\n Final L2 norm of the residuals: " + fnorm2);
-            //for (i=1;i<a.length;i++) {
-            //System.out.println(a[i]);
-            //}
-            /*
-             na[ic] = n;
-             ma[ic] = m;
-             nf[ic] = lmdiftest.nfev;
-             na[ic] = info[1];
-
-             fnm[ic] = fnorm2;
-
-             System.out.print("\n Initial L2 norm of the residuals: " + fnorm1 +
-             "\n Final L2 norm of the residuals: " + fnorm2 +
-             "\n Number of function evaluations: " + nf[ic] +
-             "\n Number of Jacobian evaluations: " + nj[ic] +
-             "\n Info value: " + info[1] +
-             "\n Final approximate solution: \n\n");
-             */
```

Comment:
removed this whole commented out section, not just the System.out, since this method doesn't even have these variables, possibly a copy paste from Lmder_f77.doMin and they werent needed?